simulation/traffic_simulator/include/traffic_simulator/entity/pedestrian_entity.hpp

```diff
@@ -107,6 +107,10 @@ class PedestrianEntity : public EntityBase
 
   void requestAssignRoute(const std::vector<geometry_msgs::msg::Pose> &) override;
 
+  void requestSpeedChange(
+    const double target_speed, const SpeedChangeTransition transition,
+    const SpeedChangeConstraint constaint, const bool continuous) override;
```

Comment:
`constaint` => `constraint`

Reply:
fixed in https://github.com/tier4/scenario_simulator_v2/pull/648/commits/b608d5e2c700eea9af558ed92a2a6d2699994108